spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -58,5 +58,56 @@
         )
       end
     end
+
+    it "gets the title tesim" do
+      expect(result["title_teim"]).to include(
+        "Harold B. Hoskins Papers"
+      )
+      expect(result["title_teim"]).to eq(
+        result["title_ssm"]
+      )
+    end
+
+    it "asserts that title filing si field is missing" do
```

Comment:
I think we should delete this test. If we have another record that does populate this field we can add a test for the field at that time.